chain/types/message_fuzz.go

```diff
@@ -34,6 +35,9 @@ func FuzzMessageDecoder(data []byte) int {
 }
 
 func FuzzAddressEncoder(data []byte) int {
+	if len(data) == 0 {
+		return -2
```

Comment:
@Kubuxu `-2` is not a valid return value, see https://github.com/dvyukov/go-fuzz#usage

Reply:
makes sense, dunno why I used it.